ghost/traffic-analytics/tinybird/scripts/unsafe_delete_all_mvs.sh

```
@@ -0,0 +1,62 @@
+set -euo pipefail
+
+# Install jq if not installed
+if ! command -v jq &> /dev/null
+then
+    echo "jq could not be found, installing..."
+    apt-get update && apt-get install -y jq
+fi
+
+# Check if --force flag is provided
+force=false
+if [[ "$@" == *"--force"* ]]; then
```

Shellcheck (0.10.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)